src/api/common/schemas/settings.json

```diff
@@ -12,8 +12,18 @@
     "noFreeze": {"type": "boolean"},
     "globalFreeze": {"type": "boolean"},
     "defaultRipple": {"type": "boolean"},
-    "emailHash": {"$ref": "hash128"},
-    "walletLocator": {"$ref": "hash256"},
+    "emailHash": {
+      "oneOf": [
+        {"type": "string", "maxLength": 0},
```

Comment:
Yes, that would be good - right now you can't clear integer fields - but I don't know how schema validator will react.